frontend/src/components/elements/CodeBlock/CodeBlock.tsx

```diff
@@ -18,6 +18,7 @@
 import Prism, { Grammar } from "prismjs"
 import React, { ReactElement } from "react"
 import { logWarning } from "lib/log"
+import _ from "lodash"
```

Comment:
nit: I think we can get away without having to import `lodash` here since `value` below is a string so is already falsy when empty, so
```javascript
value && /* ... */
```
should work